keps/sig-architecture/3000-artifact-distribution/kep.yaml

```diff
@@ -0,0 +1,19 @@
+title: Artifact Promotion and Distribution Policy and Procedure
+kep-number: NNNN
+authors:
+  - "@hh"
+owning-sig: sig-architecture
```

Comment:
Out of curiosity, why sig-release is not the owner of this ?

Reply:
ref: https://github.com/kubernetes/enhancements/pull/3079#pullrequestreview-825638483